test/unit/core/sessions.test.js

```diff
@@ -1,13 +1,22 @@
 'use strict';
 
 const mock = require('../../tools/mock');
+const BSON = require('bson');
```

Comment:
```suggestion
const BSON = require('../../../src/bson');
```
Prefer importing bson from the driver so that the BSON-EXT logic (or anything else) is tested automatically.

Reply:
good call out, though all test files except for the bson import one require it directly - any thoughts on that?

Reply:
Hm, seems like something we should fix. In test/.eslintrc.json you can add bson to the forbidden modules, but there's 24 instances and at least two are intentionally testing bson vs bson-ext, so I guess lets defer to another ticket?

Reply:
yeah, feels a bit out of scope, can file a simple clean up ticket in the test epic; would you rather I do use the internal reference for this test though?

Reply:
Totally, I don't imagine there being any impact here so whichever for now and we'll clean them up in that ticket.